sample_apps/rust/examples/query_sample.rs

```diff
@@ -0,0 +1,220 @@
+use aws_sdk_timestreamquery as timestream_query;
+use clap::Parser;
+use std::error::Error;
+use std::fs;
+pub mod utils;
+use crate::utils::query_common;
+
+async fn execute_sample_queries(
+    region: String,
+    database_name: String,
+    table_name: String,
```

Comment:
You can probably take these all as `&str` instead to avoid taking ownership of the Strings. It's a bit more flexible of an API that way.

Reply:
Changed to `&str`.